tool/gravity/cli/register.go

```diff
@@ -576,6 +576,13 @@ func RegisterCommands(app *kingpin.Application) *Application {
 	g.SystemDisablePromiscModeCmd.CmdClause = g.SystemCmd.Command("disable-promisc-mode", "Remove promiscuous mode flag and deduplication rules").Hidden()
 	g.SystemDisablePromiscModeCmd.Iface = g.SystemDisablePromiscModeCmd.Arg("name", "Name of the interface (i.e. docker0)").Default(defaults.DockerBridge).String()
 
+	g.SystemExportRuntimeJournalCmd.CmdClause = g.SystemCmd.Command("export-runtime-journal", "Export runtime journal logs to a file").Hidden()
+	g.SystemExportRuntimeJournalCmd.Package = Locator(g.SystemExportRuntimeJournalCmd.Arg("pkg", "Runtime package").Required())
```

Comment:
Runtime package can probably be auto-detected if not specified, no? By installed/runtime labels. Wouldn't it make this command easier to use?